keyring/backend.py

```diff
@@ -196,7 +196,14 @@ def _load_plugins():
 
     `initialize_func` is optional, but will be invoked if callable.
     """
-    entry_points = metadata.entry_points()['keyring.backends']
+    try: entry_points = metadata.entry_points()['keyring.backends']
```

Comment:
I think it would make sense to follow rest of the project and do try/catch using the "standard" syntax. Please change the code to
```
try:
    entry_points = metadata.entry_points()['keyring.backends']
```

Reply:
Even better, there are suggestions on the CI failure page about what needs to be changed for the CI to pass https://ci.appveyor.com/project/jaraco/keyring/builds/29440604/job/r9f96fw8fh5a2f88 -- Kindly change the code as suggested. (Note: this is not a review from project maintainers).

Reply:
Thanks. Now I have got CI pass.